src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -0,0 +1,499 @@
+
+These certificates should be signed by the same CA etcd is using to operate.
+
+How to generate these certificates is out of the scope of this documentation, as it will vary greatly between different Kubernetes distribution. Please refer to your distribution's documentation to see how to fetch the required etcd peer certificates. In Kubeadm, for example, they can be found in `/etc/kubernetes/pki/etcd/peer.{crt,key}` in the master node.
+
+[//]: # (TODO: This should be a TLS secret when https://github.com/newrelic/nri-kubernetes/issues/342 is fixed.)
+
+Once you have located or generated the etcd peer certificates, you should rename the files to match the keys we expect to be present in the secret, and create the secret in the cluster
+
+```shell
+mv peer.crt cert
+mv peer.key key
+mv ca.crt cacert
+
+kubectl -n newrelic create secret generic newrelic-etcd-tls-secret --from-file=./cert --from-file=./key --from-file=./cacert
+```
+
+Finally, you can input the secret name (`newrelic-etcd-tls-secret`) and namespace (`newrelic`) in the config snippet shown at the beginning of this section. Remember that the Helm Chart will automatically parse this config and create an RBAC role to grant access to this specific secret and namespace for the `nrk8s-controlplane` component, so there's no manual action needed in that regard.
+
+### Static endpoints [#static-endpoints]
+
+While autodiscovery should cover cases where the control plane lives inside the Kubernetes clusters, some distributions or sophisticated Kubernetes environments run the control plane elsewhere, for a variety of reasons including availability or resource isolation.
+
+For these cases, the integration can be configured to scrape an arbitrary, fixed URL regardless of whether a pod with a control plane label is found in the node. This is done by specifying a `staticEndpoint` entry. For example, one for an external etcd instance would look like this:
+
+```yaml
+controlPlane:
+  etcd:
+    staticEndpoint:
+      url: https://url:port
+      insecureSkipVerify: true
+      auth: {}
+```
+
+`staticEndpoint` is the same type of entry as `endpoints` in the `autodiscover` entry, whose fields are described above. The authentication mechanisms and schemas are supported here.
+
+Please keep in mind that if `staticEndpoint` is set, the `autodiscover` section will be ignored in its entirety.
+
+#### Limitations [#static-endpoints-limitations]
+
+<Callout variant="important">
+  If you are using `staticEndpoint`, you must change `controlPlane.kind` from `DaemonSet` to `Deployment`.
+</Callout>
+
+When using `staticEndpoint`, all `nrk8s-controlplane` pods will attempt to reach and scrape said endpoint. This means that, if `nrk8s-controlplane` is a DaemonSet (the default), your components will get scraped multiple times, potentially leading to duplicate metrics and increased billable usage. If you are using `staticEndpoint`, make sure to change `controlPlane.kind` to Deployment.
```

Comment:
unless the static endpoint have localHost. The question could be why having localhost? Since in rancher they are running outside the cluster and auto discovery does not work

Reply:
Reworded above.